### ISSUE TITLE
Add a function that checks if all the existing files are listed in the DB

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/sct_testing_management#35]

**Comment by perone** [comment]
@jcohenadad @fperdigon what I'm planning is to render an HTML report that will contain the information about the issues that tool found, separated into warning (small issues like empty demographics, etc), errors (file doesn't exist, etc), for every item in the database. I'll also do that in a way that it will be easier to add more checks in the future so we can keep improving checks as needed according to new necessities.
In summary, the tool will connect into the web management system, get the JSON, run all checks and generate a html report.

Does that sound ok ?

**Comment by jcohenadad** [comment]
@perone that sounds fantastic! where do you think we should put those scripts? under `utility/`? or create a new `scripts/` folder?
@fperdigon where did you put your API scripts?
thanks!

**Comment by perone** [comment]
I'll do a python installable package, that can be installed with `pip install -e .`, so it's easy for installing dependencies, reuse and importing. I can put it inside a directory in this repository or create another repository.

**Comment by jcohenadad** [comment]
maybe the same repo? there is already 50+ repos under neuropoly org, i have hard time remembering what this is all about ;-)

**Comment by perone** [comment]
I think it's better also, I'll use this same repo so we'll have everything integrated.

**Comment by fperdigon** [comment]
@perone  Great idea! HTML report sounds great.
@jcohenadad the  API Python client is [here]( https://github.com/neuropoly/sct_testing_management/tree/API_Python_Client_by_Fco/API_Python_Client)
I also agree in use this same repo

**Comment by perone** [comment]
@jcohenadad the path in duke for the database files is `[duke]/sct_testing/large`, correct ? Just to confirm.
